core/src/main/java/org/jsmart/zerocode/core/engine/mocker/RestEndPointMocker.java

```diff
@@ -25,10 +28,23 @@
 
     public static WireMockServer wireMockServer;
 
+    public static Boolean buildUrlPathEqualToPattern_FeatureFlag = true;
+
+    private static boolean hasMoreThanOneStubForSameUrl(List<String> urls) {
```

Comment:
@MrXavier 
I think, We won't have two exact urls. Here I think you should be calculating if we have two URLs, that have same PATH.
for instance:
url1 = /google?param1=value1
url2 = /google?param2=value2
In this case we have 2 URLs, with same path so we should match strictly. Instead of comparing URLs, compare their respective paths. If you meant smth else please let me know.